keras_nlp/models/bart/bart_seq_2_seq_lm.py

```diff
@@ -97,12 +108,8 @@ class BartSeq2SeqLM(GenerativeTask):
     # "The quick brown fox", and the decoder inputs to "The fast". Use
     # `"padding_mask"` to indicate values that should not be overridden.
     prompt = {
-        "encoder_token_ids": tf.constant([[0, 133, 2119, 6219, 23602, 2, 1, 1]]),
```

Comment:
Is this doc removal intentional? Seems like maybe the intent was to just s/tf.constant/ops.array or similar?

Reply:
Good catch and not intentional!